index.html

```diff
@@ -3177,7 +3167,7 @@ <h3>Production</h3>
       64                             # text(4)
          74797065                    # "type"
       78 1A                          # text(26)
-         45643235353139566572696669636174696F6E4B657932303138 # "Ed25519VerificationKey2018"
+         45643235353139566572696669636174696F6E4B657932303138 # "Ed25519VerificationKey2020"
```

Comment:
The CBOR bytes also needs to be updated, but I guess that can happen in a separate PR (in the new CBOR Note repo).